omero/users/command-line-interface.txt

```diff
@@ -127,20 +127,56 @@ To import a file image.tiff, use::
 
     $ bin/omero import image.tiff
 
-Many options can be passed to the :omerocmd:`import`. They can be listed using
-the :option:`-h` option::
+The options available to the import command are:
 
-    $ bin/omero import -h
+.. option:: -h, --help
 
-The :option:`-f` option displays all the used files for importing then exits::
+    Display the help for the import command
 
-    $ bin/omero import -f image.tif
-    $ bin/omero import -f images_folder
+.. option:: -f
 
-The :option:`--depth` option sets the number of directories to scan down for
-files (default: 4)::
+    Display all the used files for importing then exits::
 
-    $ bin/omero import --depth 1 images_folder
+        $ bin/omero import -f image.tif
+        $ bin/omero import -f images_folder
+
+.. option:: --depth DEPTH
+
+    Set the number of directories to scan down for files (default: 4)::
+
+        $ bin/omero import --depth 1 images_folder
+
+.. option:: --skip [SKIP [SKIP]...]
+
+    Specify optional import steps to skip. Multiple steps can be skipped and
+    multiple arguments can be supplied. Available options that can be skipped
+    are currently:
+
+    all
+        Skip all optional steps described below
+    checksum
+        Skip all steps calculation checksums either on files after transfer
+        or on pixels
+    minmax
+        Skip computation of the minima and maxima pixel values
+    thumbnails
+        Skip generation of thumbnails
+    upgrade
+        Skip upgrade check for Bio-Formats
+
+    Example of usage::
+
+        $ bin/omero import large_image --skip all
+        $ bin/omero import large_image --skip checksum --skip minmax
```

Comment:
The `--skip SKIP1 --skip SKIP2` example isn't supported by the option given above which implies `--skip SKIP1 SKIP2`